utils/python-lint

```diff
@@ -1,16 +1,26 @@
 #!/bin/bash
+# This script is a linter for Python files in the Swift codebase.
+# If there are any linting errors, the script exits with code 1; otherwise
+# it exists with retcode 0.
+#
+# Optionally, you can supply the name of one or more files (relative to the
+# root of the codebase) to limit the linting to those files.
```

Comment:
Can you add a note here that this is the script used by @swift-ci please Python lint.

That is the actual purpose of this script: to make it really easy to reproduce how @swift-ci lints so one can check for failures on ones own machine.